### PR TITLE
fixes #59

### LINKED ISSUE #59: Can we use previous Sensor Obs schemas and patterns
In the Sensor Web Enablement and SensorThings API groups and mailing lists, there was concern expressed that there was a lack of acknowledgement of previous OGC work covering data structures, metadata and queries relevant to EDR. In particular:
1. [Sensor Observation Services and its schemas](https://www.ogc.org/standards/sos) 
1. [WaterML](https://www.ogc.org/standards/waterml) 
1. and its direct derivative [TimeSeriesML](https://www.ogc.org/standards/tsml)  

SOS has XML schemas with filter patterns for time and space. Can we use these? If so, then we can add a reference to the standard.

### PR DESCRIPTION
**Pull request by dblodgett-usgs**
Please don't quibble with my language, I'm sure it is not right, please fix it.

### CODE REVIEW

**Comment by chris-little** [comment]
Good words. A useful insight provided by @m-burgoyne is that current OGC standards take features (or coverages) a first class citizens with spatio-temporal attributes. EDR API can be viewed as having the spatio-temporal coordinates as first class citizens with data as the attributes

**Comment by dblodgett-usgs** [comment]
My "use-centric" is coded language for that point. Not sure which way to put it is more relatable.

**Comment by dr-shorthair** [comment]
I would say that _coverages_ are already the spatio-temporal view. This contrasts with _features_ which are an identity-oriented view. 

Features are (usually) thought of as discrete, identifiable objects, probably typed/classified, with characteristic properties linked to the feature-type. 

OTOH Coverages deal with the distribution of one or more properties across a spatio-temporal domain. Queries on coverages are usually to get a subset of properties across a section of the spatio-temporal domain. 

Switching between these viewpoint is related to sampling and signal detection. 

What you are doing here with EDR is a streamlined query into a coverage (data-cube).

**Comment by dblodgett-usgs** [comment]
OK, so we all agree with our own words. Fine by me. I'm not sure if your comment is suggesting a change or just chiming in?

File: candidate-standard/clause_0_front_material.adoc [hunk 1]
```diff
@@ -4,7 +4,9 @@
...
+
+EDR API query patterns can be thought of as discrete sampling geometries, conceptually in-line with the feature of interest in the https://www.ogc.org/standards/sos[Sensor Observation Service (SOS)]. A typical EDR data resource is a multidimensional dataset that could be provided via a coverage API such as the http://www.ogc.org/standards/wcs[Web Coverage Service (WCS)]. In contrast to SOS and WCS, EDR implements the technical baseline of the OGC API family of standards and aims to provide a single set of use-centric query patterns. EDR query patterns range from location-centric real or virtual observation use cases to sub-setting 4-dimensional data cubes along user-supplied sampling geometries. These query-pattern building blocks do not attempt to satisfy the full scope of either SOS or WCS, but rather, provide useful building blocks to allow composition of APIs that satisfy a wide range of environmental data use cases.
```
Comment (dr-shorthair) [inline candidate-standard/clause_0_front_material.adoc:9]: This is good contextualization.

File: candidate-standard/clause_6_overview.adoc [hunk 2]
```diff
@@ -8,7 +8,7 @@ The OGC API family of standards enable access to resources using the HTTP protoc
 . To allow service users to retrieve, in effect, transient resources over a discrete sampling geometry, created by the service in response to a standardized query pattern.
 . To provide 'building blocks' allowing the construction of more complex services
 
-The EDR API can be considered a 'Sampling API'. The query creates a discrete sampling geometry against the resource of a relatively persistent data store. The query and its response are transient resources, which could be made persistent for re-use if required.
+The EDR API can be considered a 'Sampling API'. The query creates a discrete sampling geometry against the resource of a relatively persistent data store. The query and its response are transient resources, which could be made persistent for re-use if required. The functionality provided by EDR query patterns could be realized through specific implementation of the SOS (and to some extent WCS) from the https://www.ogc.org/standards/common[OGC Web Services family of (XML-based) standards]. EDR introduces a streamlined JSON-based OGC API implementation of building blocks that could be used for many of the same use cases addressed by SOS and WCS in the past.
```
Comment (dr-shorthair) [inline candidate-standard/clause_6_overview.adoc:11]: Good.

**Comment by dr-shorthair** [comment]
These additions are good @dblodgett-usgs 
However, @chris-little you merged this too fast - if you actually want reviews from interested parties, then better to wait until we've all woken up again. This was submitted then merged all within one sleep for me.

**Comment by dblodgett-usgs** [comment]
I was a bit surprised by the quick merge too...

**Comment by chris-little** [comment]
@dr-shorthair @dblodgett-usgs Sorry about that. Keen to move forward, but we do need consensus. I like 'stream-lined'

**Pull request merged**